training/ilab-wrapper/ilab

```diff
@@ -72,14 +72,15 @@ else
         '$1 == current_user || $1 == current_uid {print $2 ":" $3}' \
         /etc/subuid)
 
-    # TODO: Handle multiple subuid ranges, for now, hard fail
-    if [[ $(wc -l <<<"$CURRENT_USER_SUBUID_RANGE") != 1 ]]; then
-        if [[ -z "$CURRENT_USER_SUBUID_RANGE" ]]; then
-            echo-err "No subuid range found for user $CURRENT_USER_NAME ($UID)"
-        else
-            echo-err "Multiple subuid ranges found for user $CURRENT_USER_NAME ($UID), this is currently unsupported"
-            echo-err "$CURRENT_USER_SUBUID_RANGE"
-        fi
+    NUMBER_OF_MATCHING_SUBUID_RANGES=$(if [[ -z "$CURRENT_USER_SUBUID_RANGE" ]]; then echo 0; else printf "%s\n" "$CURRENT_USER_SUBUID_RANGE" | wc -l; fi)
```

Comment:
We really need to start adding some functions to this script.

Doesn't
 wc -l <<< "$CURRENT_USER_SUBUID_RANGE"

Work?

Reply:
Nope, for empty CURR.. it just gives `1`

Reply:
Although tbf in this case it's not empty so I could go back to using <<<

Reply:
Updated back to `<<<`